Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM node:5-slim
```

Comment:
Would this also work with `node:10.15.1-alpine`? – or any version that works with `kongfig`.